examples/playground/panel-lite-editor.js

```diff
@@ -0,0 +1,131 @@
+const appContainer = "app-container"
+const codeEditor = "code-editor"
+
+const welcomeExample=`\
```

Comment:
Refactor the examples to an external `panel-lite-examples.json` or `panel-lite-examples.js` file.